ui/panels-plugin/src/plugins/trace-table/DataTable.tsx

```diff
@@ -173,7 +173,6 @@ export function DataTable(props: DataTableProps) {
       rows={rows}
       getRowHeight={() => 'auto'}
       getEstimatedRowHeight={() => 66}
-      autoHeight={true}
```

Comment:
@andreasgerstmayr FYI, if it can affect embedded use case => https://mui.com/x/react-data-grid/layout/#flex-parent-container

Reply:
Thanks for the note! I'll check that for the next downstream release.